src/crowdsale/CrowdSale.sol

```diff
@@ -74,18 +74,16 @@ contract CrowdSale is ReentrancyGuard {
      * @return saleId
      */
     function startSale(Sale calldata sale) public returns (uint256 saleId) {
-        //todo: use 15 minutes
-        if (sale.closingTime < block.timestamp + 2 hours) {
+        if (sale.closingTime < block.timestamp) {
             revert BadSaleDuration();
         }
 
         if (sale.auctionToken.decimals() != 18) {
             revert BadDecimals();
         }
 
-        //close to 0 cases lead to very confusing results
-        //todo: use 10^(decimals-2) to allow 0.01 ethers
-        if (sale.fundingGoal < 10 ** sale.biddingToken.decimals() || sale.salesAmount < 0.5 ether) {
+        //close to 0 cases lead to precision issues.Using 0.01 bidding tokens as minimium funding goal
+        if (sale.fundingGoal < 10 ** (sale.biddingToken.decimals() - 2) || sale.salesAmount < 0.5 ether) {
```

Comment:
What if the `biddingToken` has 2 decimals (example is `EURS`)

Reply:
test case added